src/protocol/graphics.ts

```diff
@@ -238,20 +238,28 @@ export function setupGraphics(store: UIStore) {
     const pause = ThreadFront.currentPause;
     assert(pause);
 
-    const rv = await pause.repaintGraphics();
-    if (pause === ThreadFront.currentPause && rv) {
-      const { mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
-      let { description, screenShot } = rv;
-      if (screenShot) {
-        repaintedScreenshots.set(description.hash, screenShot);
-      } else {
-        screenShot = repaintedScreenshots.get(description.hash);
-        if (!screenShot) {
-          console.error("Missing repainted screenshot", description);
-          return;
+    try {
+      const rv = await pause.repaintGraphics();
+      if (pause === ThreadFront.currentPause && rv) {
+        const { mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
+        let { description, screenShot } = rv;
+        if (screenShot) {
+          repaintedScreenshots.set(description.hash, screenShot);
+        } else {
+          screenShot = repaintedScreenshots.get(description.hash);
+          if (!screenShot) {
+            console.error("Missing repainted screenshot", description);
+            return;
+          }
         }
+        paintGraphics(screenShot, mouse);
+      }
+    } catch (e) {
+      if (e.code == 33) {
```

Comment:
I'm not sure if we're exporting protocol error constants

Reply:
Using a constant 33 is fine, but this needs a comment.